index.html

```diff
@@ -0,0 +1,135 @@
+                <li class="circle__wrapper"><a href="#" class="circle"></a></li>
+                <li class="list__wrapper"><a href="#" class="list">2015</a></li>
+            </ul>
+            <p class="signup-disc">A daily selection of privately personalized reads: no acounts or signup's required</p>
+            <div class="platform">
+                <button class="pla__btn">html</button>
+                <button class="pla__btn">css</button>
+                <button class="pla__btn">javascript</button>
+            </div>
+            <button class="see-project">See Project</button>
+        </section>
+
+        <section class="about-section">
+            <div class="details-about-me">
+                <h1 class="bold-text">About Myself</h1>
+                <p class="signup-disc">i am a software developer by profesion and i help develop many softwares, you can go ahed and check out some of my projects in my portofolio, both mobile and web based and continuing to do so. if you need help regarding software development or have a project in mind feel free to contact me.</p>
+                <h2 class="let-connect">LET'S CONNECT</h2>
+            </div>
+            <ul class="contact_us_wrapper">
+                <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-github-square"></i></a></li>
+                <li><a class="icon" href="https://www.linkedin.com/in/addisalemseifu/"><i class="fab fa-linkedin-in"></i></a></li>
+                <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-angellist"></i></a></li>
+                <li><a class="icon" href="https://twitter.com/SeifuAddisalem"><i class="fab fa-twitter"></i></a></li>
+                <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-medium-m"></i></a></li>  
+            </ul>
+            <button class="see-project">See Project</button>
+            <div class="performance">
+                <h3 class="languages-title">Languages</h3>
+                <ul class="language-details">
+                    <li class="javascript"><img src="./img/javascript.png" alt="javscript picture">Javascript</li>
+                    <li class="html"><img src="./img/html.png" alt="html picture">HTML</li>
+                    <li class="css"><img src="./img/css.png" alt="css picture">CSS</li>
+                </ul>
+                <div class="total__wrapper">
+                    <div class="platform__wrapper">
+                        <h3 class="frameworks-title">Frameworks</h3>
+                        <ul class="bar__wrapper">
+                            <li class="bar"></li>
+                            <li class="bar"></li>
+                        </ul>
+                    </div>
+                    <div class="underline"></div>
+                    <div class="platform__wrapper">
+                        <h3 class="frameworks-title">Skills</h3>
+                        <ul class="bar__wrapper">
```

Comment:
- [ ] Well done on wrapping the work section in a section with a grid display as the first reviewer suggested. However, you placed the about section inside the same section when it should be outside because Work and about are two different sections . Kindly take it out and place it after the work section To make your webpage more HTml semantically correct.